README.md

```diff
@@ -19,6 +19,8 @@ npm tst
 
 ## Local development
 
+1. Create a new repository based on the `helix-pages-starter` template and add a mountpoint in the `fstab.yaml`
+1. Add the [helix-bot](https://github.com/apps/helix-bot) to the repository
 1. Install the [Helix CLI](https://github.com/adobe/helix-cli): `npm install -g @adobe/helix-cli`
-1. Start Helix Pages Proxy: `hlx up` (opens your browser at `http://localhost:3000`)
+1. Start Helix Pages Proxy: `hlx up --pagesUrl=http://main--{repository-name}--{repository-owner}.hlx3.page` (opens your browser at `http://localhost:3000`)
```

Comment:
With https://github.com/adobe/helix-cli/issues/1848 `--pagesUrl` will become optional
```suggestion
1. Start Helix Pages Proxy: `hlx up` (opens your browser at `http://localhost:3000`)
```

Reply:
I'd like to keep `--pagesUrl` for now to prevent users from proxying to helix 2 pages. 
That should be the way forward for anyone starting a new project based on this template.